sci-astronomy/stellarium/files/stellarium-0.18.2.patch

```diff
@@ -0,0 +1,21 @@
+--- a/plugins/TelescopeControl/src/CMakeLists.txt	2018-11-08 23:08:25.339981323 +0000
```

Comment:
what is the purpose of this patch?

Reply:
Otherwise it fails to build... I put details at https://github.com/Stellarium/stellarium/pull/545

Reply:
Thanks, I see this PR was merged upstream. Please use the full commit (format-patch style) for this patch.